frontend/src/metabase/questions/containers/AddToDashboard.jsx

```diff
@@ -84,44 +84,50 @@ export default class AddToDashboard extends Component {
   render() {
     const { query, collection } = this.state;
     return (
-      <ModalContent
-        title={t`Pick a question to add`}
-        className="px4 mb4 scroll-y"
-        onClose={() => this.props.onClose()}
-      >
-        <div className="py1">
-          <div className="flex align-center">
-            {!query ? (
-              <ExpandingSearchField
-                defaultValue={query && query.q}
-                onSearch={value =>
-                  this.setState({
-                    collection: null,
-                    query: { q: value },
-                  })
-                }
-              />
-            ) : (
-              <HeaderWithBack
-                name={collection && collection.name}
-                onBack={() => this.setState({ collection: null, query: null })}
-              />
-            )}
-            {query && (
-              <div className="ml-auto flex align-center">
-                <h5>Sort by</h5>
-                <Button borderless>{t`Last modified`}</Button>
-                <Button borderless>{t`Alphabetical order`}</Button>
-              </div>
-            )}
+      <div className="ml-auto mr-auto" style={{ maxWidth: 800 }}>
```

Comment:
You can use `wrapper wrapper--trim` for the classes here and that should pretty much get at what you're trying to do without needing to add ml-auto, mr-auto and a style object, although it's a bit wider at 940px. We use that in places like the new question flow options screen, and the setup flow to constrain the width of things to a more reasonable size and I think I'd want to keep doing that in this case.

Reply:
Sweet, thanks!